src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void anotherDummyTest() {
```

Comment:
Can try another meaningful name for your test or simply remove it?

Reply:
Yup that was just a dummy test to get started using JUnit test